routemaster/middleware.py

```diff
@@ -60,13 +60,13 @@ def inner(
 
         def wrapped_start_response(
             status: str,
-            headers: Dict[str, str],
+            headers: List[Tuple[str, str]],
```

Comment:
Just double checking that this is the correct type, and it's not some multi-dict like Django uses for some things?

Reply:
Yup, this is copied from https://github.com/python/typeshed/blob/master/stdlib/_typeshed/wsgi.pyi. I'm hoping we can move towards using those actual types, but that needs a mypy upgrade (which I've also got a PR for, but there are a bunch of other things which complicate that for the moment).